scaletest/templates/scaletest-runner/Dockerfile

```diff
@@ -0,0 +1,34 @@
+# This image is used to run scaletest jobs and, although it is inside
+# the template directory, it is built separately and pushed to
+# gcr.io/coder-dev-1/scaletest-runner:latest.
```

Comment:
Self-review: I ended up keeping this file close to where it's used, however, it may be confusing if someone tries to edit it in the template editor and there are no changes.

Perhaps I should move it elsewhere?

Reply:
Well, in theory we could use our CI to build and push it...

Reply:
Agreed, that was the idea behind the "Future improvements" below. 👍🏻 IMO not the highest priority right now, though.